content/blog/2020-07-22-july-20-community-gems.md

```diff
@@ -0,0 +1,183 @@
+---
+title: July '20 Community Gems
+date: 2020-07-31
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  getting started with CML, configuring your DVC cache, and how to request a 
+  tutorial video.
+picture: 2020-07-31/Gems_July_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/july-20-community-gems/460
+tags:
+  - CML
+  - Bitbucket
+  - GCP
+  - global remote
```

Comment:
both cache and global remote sound like too specific, ad-hoc terms to my mind

I think we have Gems usually, Discord 

again, not sure about terms like Bitbucket for example

Reply:
instead we can mention this things in the descriptionLong (is it intentional that we miss it, btw?) for SEO I guess

Reply:
<img width="569" alt="Screen Shot 2020-07-31 at 1 50 01 PM" src="https://user-images.githubusercontent.com/29716724/89076222-bcf2a480-d334-11ea-9149-7c18681605c0.png">
you're totally right. how about:

```
- Discord
- Gems
- CML
- GCP 
- DVC 1.0
```
?

Reply:
Also- for some reason I thought that if we had only one description text, we should only use the "description" field, not both "description" and "descriptionLong". Is that incorrect?

Reply:
it might be, but I don't remember which way it goes // cc @rogermparent

Reply:
list of tags looks good to me

Reply:
I didn't have any explicitly defined behavior in mind when implementing `description` and `descriptionLong` in the models, but I can pretty easily add functionality to seamlessly use `description` as `descriptionLong`'s slot if `descriptionLong` isn't present.

Reply:
@rogermparent 

what about this logic:

```
{descriptionLong ? (
                <div
                  className={styles.description}
                  dangerouslySetInnerHTML={{ __html: descriptionLong }}
                />
              ) : (
                <div className={styles.description}>{description}</div>
              )}
```

for example?

Reply:
@elleobrien I think the simplest for now is just implement both?

Reply:
Ah, I hadn't checked the template yet and was looking at the data layer. You're right, the behavior is already implemented in the template.
The description also shows up when I clone the repo locally. This smells like a caching issue, but I'm not 100% sure yet.